include/triton/core/tritonbackend.h

```diff
@@ -755,6 +757,9 @@ TRITONBACKEND_DECLSPEC TRITONSERVER_Error* TRITONBACKEND_BackendExecutionPolicy(
 /// the backend's execution policy after calling
 /// TRITONBACKEND_Initialize, so to be recognized changes to the
 /// execution policy must be made in TRITONBACKEND_Initialize.
+/// Also, note that if loading a Sequence Model in the backend,
```

Comment:
Say more explicitly "if using the sequence batcher for the model" as that is exactly when this happens. Is it for both direct and oldest?

Reply:
Yes.. The behavior is for both kinds. We previously had this behavior when TRT backend was in server repo. Moving back to multiple runners for sequence batcher models because of the reasoning in the discussion here:
https://github.com/triton-inference-server/server/pull/3516#discussion_r741598340